components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -334,7 +335,32 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     }
 }
 
+pub struct ApplyFlowControl {
+    timer: Instant,
```

Comment:
here timer is not accurate to reflect the actual time used by the current thread. It would be better to use "cpu_time::ThreadTime"?

Reply:
I don't get it. Can you explain more on why `cpu_time::ThreadTime` is better than `Instant`? Note 1. there is no IO in apply thread; 2. no matter it's block by IO or busy spinning,  it should yield for fairness; 3. It's a coroutine, not necessary be always processed in the same thread.

Reply:
> I don't get it. Can you explain more on why `cpu_time::ThreadTime` is better than `Instant`? Note 1. there is no IO in apply thread; 2. no matter it's block by IO or busy spinning, it should yield for fairness; 3. It's a coroutine, not necessary be always processed in the same thread.

ThreadTime only count the time when that thread runs, so if thread is paused by OS, that time will not count. 
But as you mentioned that the task can be processed by different thread, then ThreadTime cannot be used.